modules/ocf/manifests/packages/cups.pp

```diff
@@ -1,20 +1,33 @@
 class ocf::packages::cups {
 
   # install cups
-  package { [ 'cups', 'cups-bsd' ]: }
+  package { [ 'cups', 'cups-bsd', 'cups-tea4cups' ]: }
 
   file {
     # set print server destination
     '/etc/cups/client.conf':
-      content => "ServerName printhost\nEncryption Always\n",
+      content => "ServerName localhost\nEncryption Always\n",
```

Comment:
Does this change work on whiteout?

Reply:
So if I understand it, it _is_ already working on whiteout, this PR adds a local CUPS server that then forwards on to whiteout after doing local rasterization. So it's moving the rasterization step from the script for opening PDFs to a local CUPS server.

Reply:
Yes, as @jvperrin mentioned, I'm adding a local CUPS server that forwards jobs to whiteout. If I recall correctly, whiteout has its own CUPS setup specified separately in Puppet, and does not use the Puppet class `ocf::packages::cups` at all. I recall fydai mentioning that the main machines affected by my changes are the desktops, tsunami, and supernova.

Reply:
Why should these changes affect tsunami/supernova?

Reply:
ah, that makes sense. I misunderstood what's going on here

Reply:
Opening conversation again...

@ethanhs A quick Github search shows that `ocf::packages::cups` is used by `ocf_desktop`, `ocf_admin`, and `ocf_ssh`. I had previously considered limiting my changes to the desktops only, but I figured that I can also deploy my changes to supernova and tsunami, especially if this new pipeline is better... 

(Back then, I've briefly mentioned this to fydai, and he said that having my changes apply to `ocf::packages::cups` should be fine, as opposed to separately creating a CUPS configuration for desktops only)

However, if this isn't that good of an idea, then I can change my code accordingly and move y configuration under `ocf_desktop`...

Reply:
Ah I didn't realize we are already running cups on tsunami/supernova. Thanks for the explanation, and your continued work on this!